LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 vueTableTable
```

Comment:
```suggestion
Copyright (c) 2018 José Osório, Francisco Pereira, Miguel Teixeira 
```

Reply:
Add the copyright holders instead of using the project name

Reply:
I guess the fairest way to write this is sorting the names by alphabetical order, like so:
Francisco Pereira, José Osório, Miguel Teixeira

Reply:
Agreed. :+1:

Reply:
👍